src/components/implementation/tests/unit_cbufmalloc/malloc.c

```diff
@@ -0,0 +1,53 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <assert.h>
+#include <printc.h>
+#include <cos_component.h>
+#include <cbuf_mgr.h>
+
+#define NUM 1024
+char *mptr[NUM];
+
+int
+test_malloc(void)
+{
+	int i, j;
+
+	for(i=0; i<NUM; i++) {
+        printc("Call to malloc:\n");
```

Comment:
Why is this inconsistently spaced?  Looks like this line uses spaces, and the rest use tabs.